File: Determine_Color_of_a_Chessboard_Square.cpp
class Solution {
public:
    bool squareIsWhite(string coordinates) {
        
        int num1=coordinates[0];
        int num2=coordinates[1];

        if (num1%2==0&&num2%2==0 || num1%2==1&&num2%2==1){
            return false;
        }

    else 
    return true;
    }
};

File: Determine_Color_of_a_Chessboard_Square.java
class Solution {
    public boolean squareIsWhite(String coordinates) {
        int ascii = coordinates.charAt(0) - 'a';
        int val = coordinates.charAt(1) - '0';
        int total = ascii + val;
        if(total % 2 == 0){
            return true;
        }
        else{
            return false;
        }
    }
}
